ZeeguuAPI/ZeeguuAPI/ZeeguuAPI.swift

```diff
@@ -0,0 +1,334 @@
+//
+//  ZeeguuAPI.swift
+//  ZeeguuAPI
+//
+//  Created by Jorrit Oosterhof on 28-11-15.
+//  Copyright © 2015 Jorrit Oosterhof. All rights reserved.
+//
+
+import UIKit
+
+/// This class is a gateway to the Zeeguu API. You can use the instance of this class to communicate with the Zeeguu API.
```

Comment:
"you can use the instance of this class *obtained by ZeeguuAPI.sharedAPI()" to communicate...